resources/index.html

```diff
@@ -76,6 +76,9 @@
 		* [Filmic Worlds Blog](http://filmicworlds.com/blog/)
 		* [High Dynamic Range Imaging Book](https://www.cl.cam.ac.uk/~rkm38/pdfs/mantiuk15hdri.pdf)
 		* [Tone Mapping (slides)](https://www.cl.cam.ac.uk/~rkm38/pdfs/tone_mapping.pdf)
+		
+		## Image-Based Lighting
+		* https://www.fxguide.com/fxfeatured/the-definitive-weta-digital-guide-to-ibl/
```

Comment:
Should be Markdown-ed like the other links

Reply:
Woups went too fast on that one, sorry!